var/spack/repos/builtin/packages/abinit/package.py

```diff
@@ -95,12 +100,26 @@ def configure_args(self):
         options = []
         oapp = options.append
 
+        if '+wannier90' in spec:
+            oapp('--with-wannier90-libs=-L{0}'
+                 .format(spec['wannier90'].prefix.lib + ' -lwannier -lm'))
+            oapp('--with-wannier90-incs=-I{0}'
+                 .format(spec['wannier90'].prefix.modules))
+            oapp('--with-wannier90-bins={0}'
+                 .format(spec['wannier90'].prefix.bin))
+            if '+mpi' in spec:
+                oapp('MPIFC={0}/mpifc'.format(spec['mpi'].prefix.bin))
```

Comment:
Can you move this to the same `+mpi` block as the other MPI-specific changes below?

Reply:
probably; I don't think the order of the configure args are important....